lib/galaxy/tool_util/xsd/galaxy.xsd

```diff
@@ -1537,6 +1537,20 @@ of ``type`` ``data``.</xs:documentation>
         <xs:documentation xml:lang="en">Comma separated list of tags to apply to the dataset (only works for elements of collections - e.g. ``element`` XML tags).</xs:documentation>
       </xs:annotation>
     </xs:attribute>
+    <xs:attribute name="location" type="xs:anyURI">
```

Comment:
```gxdocs:added="23.1"```

(And maybe we could entice @bernt-matthias to write a linter to check all these gxdocs:added annotations are compatible with the tool profile version? 😇)

Reply:
Is this `gxdocs:added` documented somewhere. I never have seen it before (and has been used only once so far). 

But I love the idea... as well as `gxdocs:deprecated`.

Would generalize the profile check that you added in the linting code over here https://github.com/galaxyproject/galaxy/pull/15493 .. 

But not easy:

- Galaxy code does not use the xsd at all :( .. maybe it should?
- the xsd schema validation linter seems to be implemented in planemo .. maybe we should move it to the galaxy code base?
- somehow one has to find a way to access schema element annotations for the corresponding xml element 

I would like to start with an issue ... any comments on my first thoughts?